x-pack/plugin/eql/qa/common/src/main/resources/test_queries_supported.toml

```diff
@@ -29,3 +29,28 @@ description = "test built-in modulo math functions"
 # This query give a different result with ES EQL implementation because it doesn't convert to float data types for division
 expected_event_ids = [82, 83]
 query = "file where serial_event_id / 2 == 41"
+
+# Additional EQL queries with arithmetic operations that were not part of the original EQL implementation
+[[queries]]
+expected_event_ids = [82]
+query = "file where 83 - serial_event_id == 1"
+
+[[queries]]
+expected_event_ids = [82]
+query = "file where 1 + serial_event_id == 83"
+
+[[queries]]
+expected_event_ids = [82]
+query = "file where -serial_event_id + 100 == 18"
+
+[[queries]]
+expected_event_ids = [82]
+query = "file where 2 * serial_event_id == 164"
+
+[[queries]]
+expected_event_ids = [66]
+query = "file where 66.0 / serial_event_id == 1"
+
+[[queries]]
+expected_event_ids = [82]
+query = "file where serial_event_id + ((1 + 3) * 2 / (3 - 1)) * 2 == 90 or 70 + serial_event_id < 100"
```

Comment:
I don't think this is correct.
`70 + serial_event_id < 100` should match all event IDs from 1 to 29. Why is `expected_event_ids` only `82`? Or I am missing something?

Reply:
You are absolutely right. 
Looks like something changed, event the simple query like
doesn't return all 102 records:
```
  1> [2020-04-23T19:17:13,335][INFO ][o.e.x.e.EqlActionIT      ] [test] [65.test -> file where true] before test
  1> [2020-04-23T19:17:13,346][INFO ][o.e.x.e.EqlActionIT      ] [test] [55, 59, 60, 61, 65, 66, 67, 70, 71, 72, 75, 76, 77, 81, 82, 83, 86, 87, 88, 91, 92, 95, 96]
```
looking

Reply:
nevermind .... the other events have different type

Reply:
Updated to query against `process` type that lead to more results